internal/pkg/service/buffer/store/model/model.go

```diff
@@ -62,6 +62,40 @@ type ImportConditions struct {
 	Time  time.Duration     `json:"time" validate:"min=30000000000,max=86400000000000"` // 30s-24h
 }
 
+func (c ImportConditions) ShouldImport(s CurrentImportState) (bool, string) {
+	if s.Count == 0 {
+		return false, "no data to import"
+	}
+
+	defaults := DefaultConditions()
+	if c.Count == 0 {
+		c.Count = defaults.Count
+	}
+	if c.Size == 0 {
+		c.Size = defaults.Size
+	}
+	if c.Time == 0 {
+		c.Time = defaults.Time
+	}
+
+	if s.Count >= c.Count {
+		return true, fmt.Sprintf("import count limit met, received: %d rows, limit: %d rows", s.Count, c.Count)
+	}
+	if s.Size >= c.Size {
+		return true, fmt.Sprintf("import size limit met, received: %s, limit: %s", s.Size.String(), c.Size.String())
+	}
+	if s.Time >= c.Time {
+		return true, fmt.Sprintf("import time limit met, last import at: %s, limit: %s", s.Time.String(), c.Time.String())
+	}
+	return false, "conditions not met"
+}
+
+type CurrentImportState struct {
+	Count int               `json:"count"`
+	Size  datasize.ByteSize `json:"size"`
+	Time  time.Duration     `json:"time"`
```

Comment:
Tu by mali byt 2 fields, `Now time.Time, LastImportAt time.Time`, tj. aktualny stav, v metode spravis nieco ako `Now.Since(LastImportAt) -> duration`

Reply:
mozes upravit aj hlasku `last import at: ..., now: ..., limit: ...`

Reply:
omg, jo to není timestamp 🤦‍♂️